src/screens/Settings/index.tsx

```diff
@@ -274,6 +276,10 @@ function Settings() {
               setMaxWorkers={setMaxWorkers}
               toggleDarkTrayIcon={toggleDarkTrayIcon}
               darkTrayIcon={darkTrayIcon}
+              addGamesToStartMenu={addGamesToStartMenu}
+              addDesktopShortcuts={addDesktopShortcuts}
+              toggleAddDesktopShortcuts={() => setAddDesktopShortcuts(!addDesktopShortcuts)}
```

Comment:
WE have that toggle hook for a reason, because using like this can lead to errors when writing the file if the user click several times. So use the same hook we have for the other toggles instead of using only `userState`.

Reply:
ok